### ISSUE TITLE
Issue with translation in admin

### ISSUE THREAD

**Issue by GBteamFr** [getgrav/grav-plugin-admin#2219]
Hi,

i find an issue with all translations on the administration. no more variables are translated, I only see the system names. See attachment.

Do you have any idea to resolve this issue?

Regards
![Capture d’écran 2022-01-05 101038](https://user-images.githubusercontent.com/97158191/148191467-9b661f16-5761-4baf-9e09-a46127bf6abe.jpg)

**Comment by omega8cc** [comment]
Yep, we can see the same in sites upgraded recently, after they were running probably two-years old Grav versions before.

**Comment by w00fz** [comment]
Make sure your Translations setting is enabled in the configuration:

Configuration -> System -> Languages -> Translations -> [YES]

**Comment by omega8cc** [comment]
@w00fz That helped, thanks! It was not easy to determine what is going on since entire interface had only the placeholders instead of normal labels.

**Comment by w00fz** [comment]
Apologies about that, it was one of those inevitable broken states caused by big upgrades jump.

@mahagr can we make sure the translation setting gets automatically enabled after upgrade from a very old version?
